recipe/meta.yaml

```diff
@@ -32,14 +32,17 @@ test:
     - pip
   commands:
     - pip check || true  # [not win]
-    - pi check || exit 0  # [win]
+    - pip check || (exit 1)  # [win]
```

Comment:
hmm on windows this isn't working here so well.  please use ' - pip check || cmd /K "exit /b 0"  # [win]'  instead

Reply:
Thanks. Your solution works well! I tried to find the correct command too and I thought that `pip check || (exit 1)  # [win]` can be used because if `pip check` fails  it passes

Reply:
I fixed it